zookeeper-server/src/main/java/org/apache/zookeeper/cli/CliCommand.java

```diff
@@ -72,23 +72,23 @@ public void setZk(ZooKeeper zk) {
 
     /**
      * get the string used to call this command
-     * @return
+     * @return cmdStr
```

Comment:
These don't really provide any value to the reader of the javadoc, as this variable is implementation details and are meaningless to the person reading the javadocs. These unneeded return tags should just be removed from the javadoc in these cases where there's no meaningful description in the return tag.

Reply:
Thanks, fixed.